lib/dialogue.rb

```diff
@@ -0,0 +1,39 @@
+# Dialogue provides formatted messages for responses to clients.
```

Comment:
Great idea, usually the pattern for message strings is to hold them in a yml file and require them in, something akin to this: https://www.dafyddcrosby.com/using-i18n-in-sinatra/

according to locales you can then start internationalising apps